php.ini-development

```diff
@@ -178,18 +173,19 @@
 ; http://php.net/engine
 engine = On
 
+; This directive is DEPRECATED.
 ; This directive determines whether or not PHP will recognize code between
 ; <? and ?> tags as PHP source which should be processed as such. It is
 ; generally recommended that <?php and ?> should be used and that this feature
 ; should be disabled, as enabling it may result in issues when generating XML
 ; documents, however this remains supported for backward compatibility reasons.
 ; Note that this directive does not control the <?= shorthand tag, which can be
 ; used regardless of this directive.
-; Default Value: On
+; Default Value: Off
 ; Development Value: Off
 ; Production Value: Off
 ; http://php.net/short-open-tag
-short_open_tag = Off
+;short_open_tag =
```

Comment:
The default value should still be explicitly listed here.

Reply:
And the default value / development value / production value above should also be dropped, it's enough if the default is listed here in this comment.